Source/sync.cpp

```diff
@@ -261,7 +261,7 @@ Uint32 sync_update(int pnum, const Uint8 *pbBuf)
 		app_fatal("bad sync command");
 	}
 
-	/// ASSERT: assert(gbBufferMsgs != BUFFER_PROCESS);
+	assert(gbBufferMsgs != BUFFER_PROCESS);

```

Comment:
Windows CI failing with:
> C:\projects\devilutionx\Source\multi.cpp(469): error C2065: 'BUFFER_PROCESS': undeclared identifier [C:\projects\devilutionx\devilution.vcxproj]

Reply:
```suggestion
	assert(gbBufferMsgs != 2);
```

Reply:
That explains the odd comment in multi.cpp :)